common/nicknames/wc_nicknames.txt

```diff
@@ -1,4 +1,111 @@
 ﻿###Historical
+nick_the_banshee = {}
+nick_twilight_of_the_gods = {}
+nick_the_sunblade = {}
+nick_the_vile = {}
+nick_the_burning_one = {}
+nick_the_emissary = {}
+nick_the_nightingale = {}
+nick_the_brute = {}
+nick_the_weaponmaster = {}
+nick_the_cunning = {}
+nick_the_unstoppable = {}
+nick_the_thunderous = {}
+nick_the_lightning_fist = {}
+nick_the_thunder_king = {}
+nick_the_mystery_seeker = {}
+nick_the_outcast = {}
+nick_the_redeemer = {}
+nick_the_decrepit = {}
+nick_the_blade = {}
+nick_the_corruptor = {}
+nick_the_calculator = {}
+nick_the_shatterer = {}
+nick_the_blightcaller = {}
+nick_the_summoner = {}
+nick_the_zealous = {}
+nick_the_ravager = {}
+nick_the_remorseless = {}
+nick_the_wildshaper = {}
 nick_the_prophet = {}
 nick_the_harbinger_of_the_void = {}
-nick_the_whacked = {}
\ No newline at end of file
+nick_the_ashbringer = {}
+nick_the_adamant = {}
+nick_the_australian = {}
+nick_the_lightbringer = {}
+nick_the_dire = {}
+nick_the_defiler = {}
+nick_the_sun_king = {}
+nick_lord_of_the_forest = {}
+nick_the_godslayer = {}
+nick_the_divine = {}
+nick_the_slitherer = {}
+nick_the_whacked = {}
+nick_the_darkener = {}
+
+### From Lifestyle
+nick_the_ice_hearted = {}
+nick_the_seeker = {}
+nick_the_mighty = {}
+
+### Shat'yar Historical
+nick_the_hungering = {}
+
+### Mantid paragon nicknames
+nick_the_swarmkeeper = {}
+nick_the_lucid = {}
+nick_the_locust = {}
+nick_the_manipulator = {}
+nick_the_wind_reaver = {}
+nick_the_prime = {}
+nick_the_unscathed = {}
+nick_the_bloodcaller = {}
+nick_the_deathcaller = {}
+nick_the_dissector = {}
+nick_the_bloodseeker = {}
+nick_the_poisoned_mind = {}
+
+### Murlocs Culture
+nick_the_old_murkeye = {}
+
+### Classes
+nick_the_spiritbinder = {
+	#allow = {
+	#	age = 16
+	#	OR = {
+	#		is_shaman_class_trigger = yes
+	#		is_loa_priest_class_trigger = yes
+	#	}
+	#}
+	#chance = {
+	#	factor = 1
+	#}
+}
+nick_the_thunderbringer = {
+	#allow = {
+	#	age = 16
+	#	OR = {
+	#		is_shaman_class_trigger = yes
+	#	}
+	#}
+	#chance = {
+	#	factor = 1
+	#}
+}
+
+### Titans names
+# nick_the_highfather = {}
+# nick_the_lifebinder = {}
+# nick_the_thunderer = {}
+# nick_the_shaper_of_worlds = {}
+# nick_the_lore_keeper = {}
+# nick_the_champion_of_the_pantheon = {}
+# nick_the_destroyer_worlds = {}
```

Comment:
Should be the destroyer of worlds.
```suggestion
# nick_the_destroyer_of_worlds = {}
```

Reply:
Ignore this, the localization that will be seen is correct, so this doesn't matter too much.

Reply:
This is the title of Sargeras, which is currently out of the game. So I don't think it matters right now.